tests/parser/features/test_immutable.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+
+code_list = [
+    (
+        f"""
+VALUE: immutable({typ})
+
+@external
+def __init__(_value: {typ}):
+    VALUE = _value
+
+@view
+@external
+def get_value() -> {typ}:
+    return VALUE
+    """,
+        value,
+    )
+    for typ, value in (
+        ("uint256", 42),
+        ("int256", -(2 ** 200)),
+        ("int128", -(2 ** 126)),
+        ("address", "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE"),
+        ("bytes32", b"deadbeef" * 4),
+        ("bool", True),
+    )
+]
+
+
+@pytest.mark.parametrize("code,value", code_list)
```

Comment:
It makes the pytest parametrize decorator much nicer if you just did `typ, value` instead, and constructed the code inside the test, otherwise it'll display the whole code for the test case instead of something like `test_value_is_stored_correctly[uint256-42]`

Reply:
ooo slick, that would've made the failures a lot easier to visually parse lol
Will modify for sure